cluster/gce/manifests/etcd.manifest

```diff
@@ -9,14 +9,24 @@
 "securityContext": {
     "seccompProfile": {
         "type": "RuntimeDefault"
-    }
+    },
+    "runAsUser": {{runAsUser}},
+    "runAsGroup": {{runAsGroup}}
 },
 "priorityClassName": "system-node-critical",
 "priority": 2000001000,
 "hostNetwork": true,
 "containers":[
     {
     "name": "etcd-container",
+    "securityContext": {
+      "allowPrivilegeEscalation": false,
+      "capabilities": {
+        "drop": [
+          "all"
+        ]
+      }
+    },
     "image": "{{ pillar.get('etcd_docker_repository', 'k8s.gcr.io/etcd') }}:{{ pillar.get('etcd_docker_tag', '3.4.13-0') }}",
     "resources": {
       "requests": {
```

Comment:
Online 39 we should be referring to the etcd binary with the version number. Because you are setting DO_NO_MOVE_BINARIES.

Reply:
fixed